keyboards/gboards/k/ginny/readme.md

```diff
@@ -0,0 +1,44 @@
+# Ginny!
+
+![Ginny](https://assets.bigcartel.com/product_images/247416878/IMG_20191025_1424438-01.jpeg)
+
+Quite possibly the smallest, most feature-filled board you can fit in your pocket.
+
+With multi-language onboard ASETNIOP and all sorts of fun shenanigans.
+da, de, en, en-colemak, en-dvorak, es, fn, fr, hu, it, nl, nw, pl, pt, ro, sv, tr
```

Comment:
`en-dvorak` requires host machine set to `us-qwerty`, is this intended behaviour?

Reply:
This is, it's just generated from the official dictionaries so I'm unsure how to proceed. If someone (who uses dvorak) has an issue with it, it'll be changed.